### PR TITLE
Add badge on presets menu #41

### PR DESCRIPTION
**Pull request by MIKOLAJW197**
Add new feature from #41

### CODE REVIEW

File: src/app/stage/stage/stage.component.html [hunk 1]
```diff
@@ -24,6 +24,9 @@
     aria-label="Presets"
     mat-icon-button
     matTooltip="Presets"
+    matBadge="{{ presets.length }}"
```
Comment (vitaliy-bobrov) [inline src/app/stage/stage/stage.component.html]: 1. `[matBadge]="presets.length"`
2. Won't could default - empty preset here, only custom `presets.length - 1`

File: src/app/stage/stage/stage.component.html [hunk 2]
```diff
@@ -24,6 +24,9 @@
...
+    matBadgeColor="accent"
+    [matBadgeHidden]="!presets.length"
```
Comment (vitaliy-bobrov) [inline src/app/stage/stage/stage.component.html]: presets.length < 2

File: src/app/stage/stage/stage.component.html (hunk 1 above)
Comment (MIKOLAJW197) [reply src/app/stage/stage/stage.component.html]: Are you sure with this `presets.length - 1` ? You don't want to count _Default_, do you?
Now with only `presets.length` it looks like this:
obraz

Comment (vitaliy-bobrov) [reply src/app/stage/stage/stage.component.html]: Yes, you're right, so just update binding

File: src/app/stage/stage/stage.component.html [hunk 3]
```diff
@@ -24,7 +24,7 @@
     aria-label="Presets"
     mat-icon-button
     matTooltip="Presets"
-    matBadge="{{ presets.length }}"
+    [matBadge]="presets.length | number"
```
Comment (MIKOLAJW197) [inline src/app/stage/stage/stage.component.html]: Not sure, if it good approach 😬

Comment (vitaliy-bobrov) [reply src/app/stage/stage/stage.component.html]: Ivy... ok, let it be like that for now ;)

**Pull request merged**